devel/VisualStudio/stdbool.h

```diff
@@ -13,8 +13,8 @@
   include directory as used by VisualStudio. 
 */
 
-#ifndef __STDBOOL_H__
-#define __STDBOOL_H__
+#ifndef ERT_STDBOOL_H
+#define ERT_STDBOOL_H
```

Comment:
is it actually allowed to overwrite headers of the standard library? `stdbool.h` seems to be featured by the ISO-C standard...